src/components/Text.tsx

```diff
@@ -54,7 +55,7 @@ function Text({color, fontSize = variables.fontSizeNormal, textAlign = 'left', c
 
     const isOnlyCustomEmoji = useMemo(() => {
         if (typeof children === 'string') {
-            return containsOnlyCustomEmoji(children);
+            return containsOnlyCustomEmoji(children.replace(CONST.UNICODE.LTR, ''));
```

Comment:
Fixed. MenuItem prepend LTR unicode on Android only, so we need to clean it here before checking. (we still render the whole text with the LTR unicode)

https://github.com/Expensify/App/blob/ae3b0770da4dcbe0ac963b27bc9093d8a9c45c52/src/components/MenuItem.tsx#L617